src/Codeception/SuiteManager.php

```diff
@@ -123,7 +124,7 @@ public function loadTests(string $path = null): void
     /**
      * @param TestInterface|DataProviderTestSuite $test
      */
-    protected function addToSuite($test): void
+    protected function addToSuite(\PHPUnit\Framework\Test $test): void
```

Comment:
This change looks like conflict between `@param` and type.
I think that `@param` can be removed.